README.md

```diff
@@ -349,3 +349,47 @@ To generate an authentication token run RAILS_ENV=production bin/rails generate_
 ```
 {"invoked_by" => "workflow-service"}
 ```
+
+## File layout
+Objects created prior to August 2024 using the legacy, unversioned layout. Objects created thereafter have the new versioned layout.
+
+In a future workcycle, access applications (image server, streaming server, etc.) will be changed to utilize the versioned layout, at which point all objects can be migrated to the versioned layout.
+
+### Unversioned layout
+In PURL file system:
+```
+/purl/document_cache/bc/123/df/4567/
+  |   cocina.json
+  |   public <-- public xml
+  \-- meta.json
```

Comment:
What does `\--` mean in this layout?

Reply:
The end of this directory. I took this from https://datatracker.ietf.org/doc/html/draft-kunze-bagit-10

Reply:
OK, thanks, that was new to me (but maybe not other people).